prod/westeurope/internal/api/functions_services_r3/function_app/terragrunt.hcl

```diff
@@ -118,7 +130,7 @@ inputs = {
     FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
     FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
-    SLOT_TASK_HUBNAME = "ProductionTaskHub"
+    #SLOT_TASK_HUBNAME = "ProductionTaskHub"
```

Comment:
@balanza does any slot need it's proper SLOT_TASK_HUBNAME?
The new module (3.012 ) is responsible to set the value and we are going to have Prod with its proper name and two more slots with the same name.
If required I can change the module.

Reply:
SLOT_TASK_HUBNAME is dynamic and associated with slot name.
If u set a different value for SLOT_TASK_HUBNAME it will override module preset value (this is the theory but try it with a test :) )